### ISSUE TITLE
No Longer Synching After Latest Update

### ISSUE THREAD

**Issue by THX723** [boredazfcuk/docker-icloudpd#402]
All had been working fine for a long time, until I updated to the latest drop sometime yesterday. It makes it all the way to "Generating list of files in iCloud. This may take a long time..." and then nothing happens for more than a day. No further log entries either.

```
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Generating list of files in iCloud. This may take a long time if you have a large photo collection. Please be patient. Nothing is being downloaded at this time
> -- | -- | --
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Check for new files using password stored in keyring file
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Failsafe file /home/user/iCloud/.mounted exists, continuing
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Check download directory mounted correctly...
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Keyring file exists, continuing
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Synchronisation starting at 14:10:04
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Sync user: user
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Keyring file exists, continuing
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Directory is writable: /config/python_keyring/
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Failsafe file /home/user/iCloud/.mounted exists, continuing
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Check download directory mounted correctly...
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Directory is writable: /config/python_keyring/
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Ignore Synology extended attribute directories: Disabled
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Authentication domain: com
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Downloading from: icloud.com
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Sending Webhook startup notification
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Webhook notifications enabled
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Convert HEIC to JPEG: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Skip videos: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Live photo size: original
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Stop downloading when prexisiting files count is: Download All Photos
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Downloading photos from: Download All Photos
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Number of most recently added photos to download: Download All Photos
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Skip live photos: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Skip download check: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Single pass mode: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Photo size: original
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Delete after download: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Auto delete: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Set EXIF date/time: false
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Synchronisation delay (minutes): 0
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Synchronisation interval: 86400
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Folder structure: {:%Y/%m/%d}
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Download destination directory: /home/user/iCloud
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Cookie expiry notification period: 7
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Cookie path: /config/thx723gmailcom
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Authentication Type: MFA
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Apple ID: <redacted>
> 2023/10/02 14:10:04 | stdout | 2023-10-02 14:10:04 INFO     Loading configuration from: /config/icloudpd.conf
> 2023/10/02 14:10:03 | stdout | 2023-10-02 14:10:03 INFO     Python version: 3.11.5
> 2023/10/02 14:10:03 | stdout | 2023-10-02 14:10:03 INFO     Alpine Linux 3.18.3
> 2023/10/02 14:10:03 | stdout | 2023-10-02 14:10:03 INFO     ***** For support, please go here: https://github.com/boredazfcuk/docker-icloudpd *****
> 2023/10/02 14:10:03 | stdout | 2023-10-02 14:10:03 INFO     ***** boredazfcuk/icloudpd container for icloud_photo_downloader v1.0.651 started *****
> 
```

**Comment by boredazfcuk** [comment]
There was a logging visibility issue introduced by an upstream package, but I believe that was fixed in v1.0.651. It would still complete if left for long enough though, not just sit where your log ends. Latest version is 657 so it would be worth moving to that one, just in case.

Also check you can log into icloud.com without being prompted for additional info, or to agree to a new change in terms of service.

...and make sure Advanced Data Protection is disabled.

**Comment by THX723** [comment]
Okay updating to 657 seemed to have resolved the issue. Looks like 651 was definitely funky. FYI, I verified there were no issues with iCloud.com credential and ADP has always been disabled.
